src/harness/reference_models/examples/benchmark/time_dpa.py

```diff
@@ -114,23 +114,25 @@ def PrepareSimulation():
   zone_catb = dpa_zone.buffer(extend_catb_deg).intersection(us_border)
 
   if urban_areas is not None:
+    # simplify the huge urban_areas for quicker inclusion tests
     urban_areas = urban_areas.intersection(zone_catb)
 
   # - Distribute the CBSDs
+  print ' - Cat A indoor'
   cbsds_cat_a_indoor = entities.GenerateCbsdsInPolygon(
       num_sites * ratio_cat_a_indoor,
```

Comment:
Maybe use of 2 Space Indentation instead of 4 here and below

Reply:
Indentation in continuing statement (after "(") is auto done to 4 char, independently of the 2 char of global indentation. I think that is the common way to use, and as supported by most editors python modes